docusaurus/docs/engineering/boba-backend/APIs/intro.md

```diff
@@ -4,23 +4,56 @@ sidebar_position: 1
 
 # Introduction to APIs
 
-APIs allow applications (clients) to interact with another application (server) without unfettered access to the underlying data, or to the whole set of operations that can be executed on it. In short, an API is a contract defined by the server that establishes a common language that clients can use to communicate with it. Most operations allowed by APIs boil down to fetching, storing or updating data.
+Application Programming Interfaces (APIs) allow applications (clients) to request data or actions from other applications (servers), without being given full access to all the data and actions that exists on those servers.
 
-:::note
+![Example of client/server HTTP request/response exchange](./client-server.excalidraw)
+
+In short, **a server's API defines the language ("contract") that client applications must use to communicate with it**. Most API operations involve fetching, storing, or updating data.
```

Comment:
I don't know that the word "contract" really clarifies anything here for me. Does the API define the common language in the sense that it says "your requests to the server must fit these requirements/be in this specific format"? And "contract" then implies in my head the give and take of "you give me this and I give you that" but I'm not sure

Reply:
> A REST API also specifies the format of the data the client needs to send for the operation and the format of the data it will receive in response.

I guess this is kinda clarified below